src/udp_wrap.cc

```diff
@@ -497,9 +537,25 @@ void UDPWrap::DoSend(const FunctionCallbackInfo<Value>& args, int family) {
     }
   }
 
-  uv_buf_t* bufs_ptr = *bufs;
-  if (err == 0 && !UNLIKELY(env->options()->test_udp_no_try_send)) {
-    err = uv_udp_try_send(&wrap->handle_, bufs_ptr, count, addr);
+  if (err == 0) {
+    err = wrap->Send(*bufs, count, addr);
+  }
```

Comment:
No braces. Even easier to read/grep when on a single line.

Reply:
Removig the braces but prefer to keep the `err = ...` on the next line